src/linear_algebra/orientation.lean

```diff
@@ -47,28 +47,30 @@ is equivalent to `mul_action.orbit_rel`. -/
 def same_ray (v₁ v₂ : M) : Prop :=
 ∃ (r₁ r₂ : R), 0 < r₁ ∧ 0 < r₂ ∧ r₁ • v₁ = r₂ • v₂
 
-variables (M)
+variables {R}
+
+/-- `same_ray` is reflexive. -/
+@[refl] lemma same_ray.refl [nontrivial R] (x : M) : same_ray R x x :=
+⟨1, 1, zero_lt_one, zero_lt_one, rfl⟩
 
 /-- `same_ray` is symmetric. -/
-lemma symmetric_same_ray : symmetric (same_ray R : M → M → Prop) :=
-λ _ _ ⟨r₁, r₂, hr₁, hr₂, h⟩, ⟨r₂, r₁, hr₂, hr₁, h.symm⟩
+@[symm] lemma same_ray.symm {x y : M} : same_ray R x y → same_ray R y x :=
+λ ⟨r₁, r₂, hr₁, hr₂, h⟩, ⟨r₂, r₁, hr₂, hr₁, h.symm⟩
 
 /-- `same_ray` is transitive. -/
-lemma transitive_same_ray :
-  transitive (same_ray R : M → M → Prop) :=
-λ _ _ _ ⟨r₁, r₂, hr₁, hr₂, h₁⟩ ⟨r₃, r₄, hr₃, hr₄, h₂⟩,
+@[trans] lemma same_ray.trans {x y z : M} : same_ray R x y → same_ray R y z → same_ray R x z :=
+λ ⟨r₁, r₂, hr₁, hr₂, h₁⟩ ⟨r₃, r₄, hr₃, hr₄, h₂⟩,
   ⟨r₃ * r₁, r₂ * r₄, mul_pos hr₃ hr₁, mul_pos hr₂ hr₄,
    by rw [mul_smul, mul_smul, h₁, ←h₂, smul_comm]⟩
 
-/-- `same_ray` is reflexive. -/
-lemma reflexive_same_ray [nontrivial R] :
-  reflexive (same_ray R : M → M → Prop) :=
-λ _, ⟨1, 1, zero_lt_one, zero_lt_one, rfl⟩
+@[symm] lemma same_ray_comm {x y : M} : same_ray R x y ↔ same_ray R y x :=
```

Comment:
Should this also be tagged `@[symm]`? I thought only `same_ray.symm` needed it.

Reply:
Whoops, that's a mistake

Reply:
```suggestion
lemma same_ray_comm {x y : M} : same_ray R x y ↔ same_ray R y x :=
```